src/Tizen.Multimedia.Vision/MediaVision/InferenceModelConfiguration.cs

```diff
@@ -46,7 +46,7 @@ public class InferenceModelConfiguration : EngineConfiguration
         private const string _keyModelStdValue = "MV_INFERENCE_MODEL_STD_VALUE";
         private const string _keyBackendType = "MV_INFERENCE_BACKEND_TYPE";
         private const string _keyTargetType = "MV_INFERENCE_TARGET_TYPE";
-        private const string _keyTargetDevice = "MV_INFERENCE_TARGET_DEVICE";
+        private const string _keyTargetDevice = "MV_INFERENCE_TARGET_DEVICE_TYPE";
```

Comment:
Don't we need to change `_keyTargetDevice` to `_keyTargetDeviceType`?

Reply:
Thank you the review. I fixed it to _keyTargetDeviceType.